doc/api/api_changes/2017-12-06-KL.rst

```diff
@@ -0,0 +1,13 @@
+Change return value of Axes.get_shared_[x,y,z]_axes()
+-----------------------------------------------
+
+The return of get_shared_x_axes() and respectively for y and z axis
+returned a Grouper object. Now it returns a WeakSet object.
```

Comment:
The method `matplotlib.Axes.get_shared_x_axes` (and y and z) used to return `~.cbook.Grouper` objects.  Now it returns a `~.weakref.WeakSet` object.